server/api/api.go

```diff
@@ -0,0 +1,129 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/mattermost/mattermost-plugin-autolink/server/link"
+)
+
+const PluginIDContextValue = "pluginid"
+
+type LinkStore interface {
```

Comment:
Does this belong here?

Reply:
The API is defining what it requires. I think it can go here. Is there a better place?

Reply:
Good point, not that I can think of.